public/_posts/_articles/yearn-finance/yvboost/es.md

```diff
@@ -0,0 +1,99 @@
+
+## La yVault yvBOOST 
+
+![](image3.png?w=128&h=128)
+
+### Gana CRV compuesto con impulso
+
+El yVault yvBOOST es una versión compuesta y completamente automatizada de la yVault yveCRV explicada anteriormente.
+
+En pocas palabras, esta vault reclama tus recompensas semanales de 3CRV automáticamente y las usa para adquirir más yveCRV (ya sea mediante compra en el mercado o acuñando, según cuál sea más eficiente en el momento de la cosecha).
+
+Una vez depositados, al igual que en la yVault yveCRV, Yearn maneja y optimiza el poder de voto de tus tokens CRV. No necesitas preocuparte por reclamar las tarifas de protocolo semanales de [Curve.fi](http://curve.fi/), ¡la vault lo hace por ti!
+
+¡Esta es una vault de "establecer y olvidar" donde tus tokens CRV crecen exponencialmente, aprovechando el poder del interés compuesto!
+
+Ahora quizás te preguntes cómo se extraen las ganancias obtenidas de tus tokens CRV en la vault, cuando, como se mencionó anteriormente, cualquier CRV depositado en yveCRV o yvBOOST está bloqueado. Si bien no puede retirarse de la vault de yveCRV, en realidad puedes intercambiar ambos tokens de la vault en Sushiswap. Esto se debe a que Yearn y sus usuarios brindan liquidez en Sushiswap para permitir el intercambio de tus tokens yveCRV e yvBOOST por ETH (o cualquier cosa, en realidad).
+
+### Un poco de alfa.
+
+Yearn compra yvBOOST del mercado, lo desenvuelve en yveCRV y dona ese yveCRV a la vault de yvBOOST, lo que aumenta el valor subyacente de yvBOOST.
+
+## yVaults de Yearn para quien tiene CRV en Staking
+
+Hacer staking de tu CRV directamente en la plataforma [Curve.fi](http://curve.fi/) significa bloquear tu token CRV a cambio de un token veCRV no transferible que te permite reclamar manualmente una parte de la tarifa del protocolo (3CRV). Puedes usar este token veCRV para re-equilibrar manualmente tus votos y obtener un impulso en tu liquidez proporcionada a la plataforma [Curve.fi](http://curve.fi/).
+
+## Vaults yveCRV y yveBOOST de Yearn
+
+Ambas vaults de Yearn recompensan a quienes hacen staking de CRV con una parte del CRV bloqueado por Yearn, lo que lo convierte en un destino ideal para aquellos que desean hacer staking de CRV sin dejar de tener liquidez:
+
+-   Gana una parte de las tarifas comerciales del protocolo (3Crv) de [Curve.fi](http://curve.fi/), reinvertidas automáticamente (para la vault yvBOOST).
+-   Gana una parte del reclamo de Yearn de las tarifas de protocolo de [Curve.fi](http://curve.fi/), además de tus propias recompensas (¡más 3CRV!), reinvertidas automáticamente (para la vault yvBOOST).
+-   El poder de voto colectivo de los tokens veCRV se optimiza y re-equilibra automáticamente para maximizar las recompensas en todas las vaults de grupo de Yearn en Curve.
```

Comment:
AS-IS: -   El poder de voto colectivo de los tokens veCRV se optimiza y re-equilibra automáticamente para maximizar las recompensas en todas las vaults de grupo de Yearn en Curve.
TO-BE: -   El poder de voto colectivo de los tokens veCRV se optimiza y re-equilibra automáticamente para maximizar las recompensas en todas las vaults de Yearn que usan tokens de fondos de Curve.

Reply:
Done